components/Village/Detail/index.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div class="village-detail">
+    <div class="village-detail__action">
+      <BaseButton variant="primary" @click="$router.go(-1)">
+        <template #icon>
+          <div class="village-detail__action-button">
+            <div class="village-detail__action-icon">
+              <!-- eslint-disable-next-line vue/no-v-html -->
+              <div v-html="require('~/assets/icons/arrow-left.svg?raw')" />
```

Comment:
friendly discussion:

Hello @yoslie I just wondering why use `v-html` to render icon? Although I think it's fine because it's not user-generated content maybe there's another approach to this SVG icon? 

the official Vue.js docs state something like this:
`If you find yourself trying to compose templates using v-html, try to rethink the solution by using components instead.`

ref: https://vuejs.org/api/built-in-directives.html#v-html

Reply:
sure thanks for sharing your thoughts 👍